src/electionguard/decryption.py

```diff
@@ -81,7 +88,7 @@ def compute_decryption_share(
 
 
 def compute_compensated_decryption_share(
-    guardian_key: ElectionPublicKey,
+    guardian_keys: ElectionKeyPair,
```

Comment:
nitpick: `key_pair` should be used. The use of `keys` is confusing and it happens throughout the code. It immediately creates an issue the minute we combine `keys` like for an `ElectionKey`. 

Could be `guardian_key_pair` could also just be `key_pair`. I know it's avoided because of the duplicate `key_pair` within the object.